delta/rdf/src/main/scala/ch/epfl/bluebrain/nexus/delta/rdf/jsonld/api/JsonLdJavaApi.scala

```diff
@@ -63,14 +63,16 @@ object JsonLdJavaApi extends JsonLdApi {
 
   // TODO: Right now this step has to be done from a JSON-LD expanded document,
   // since the DocumentLoader cannot be passed to Jena yet: https://issues.apache.org/jira/browse/JENA-1959
-  override private[rdf] def toRdf(input: Json)(implicit opts: JsonLdOptions): Either[RdfError, Model] = {
-    val model       = ModelFactory.createDefaultModel()
+  override private[rdf] def toRdf(input: Json)(implicit opts: JsonLdOptions): Either[RdfError, DatasetGraph] = {
```

Comment:
JENA-1959 has been marked as Fixed/Done for 3.17.0, should we resolve this as well?

Reply:
I made that fix myself. However I still feel we should first expand and then convert to a graph. There are some tricky details about it I don't feel like trying out now.

Reply:
Ok... maybe we can revisit this in the future.